spec/features/issues_spec.rb

```diff
@@ -172,13 +172,13 @@
       it 'sorts by recently due milestone' do
         visit project_issues_path(project, sort: sort_value_milestone_soon)
 
-        first_issue.should include("foo")
+        expect(first_issue).to include("foo")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.